Segments/Blackburrow.mapproj

```diff
@@ -269807,6 +269826,193 @@ return yima;
       <script name="OnIncomingPlayer" enabled="false">
         <block><![CDATA[]]></block>
         <block><![CDATA[
+]]></block>
+        <block><![CDATA[]]></block>
+      </script>
+    </entity>
+    <entity name="BossNoteableLevel18InsaneWiz">
+      <script name="OnSpawn" enabled="true">
+        <block><![CDATA[]]></block>
+        <block><![CDATA[
+var insaneWiz = new Humanoid()
+    {
+        Name = "Insane Froglok Wizard", Alignment = Alignment.Chaotic,
+        MaxHealth = 4500, Health = 4500,
+        BaseDodge = 40,
+        Body = 453,
+        CanSwim = true,
+    
+        Experience = 250000,
+        
+        BasePenetration = ShieldPenetration.Heavy,
+        HideDetection = 45,
+
+        Immunity  =  CreatureImmunity.Poison,
+        
+        
+        VisibilityDistance = 3,
+          
+    };
+    
+    insaneWiz.Stats[EntityStat.FireProtection].Base = 100;
+    insaneWiz.Stats[EntityStat.IceProtection].Base = 75;
+    insaneWiz.Stats[EntityStat.LightningResistance].Base = 75;
+    insaneWiz.AddImmunity(typeof(WhirlwindSpell));
+    insaneWiz.AddImmunity(typeof(PoisonCloudSpell));
+   
+    insaneWiz.AddStatus(new NightVisionStatus(insaneWiz));
+    insaneWiz.AddStatus(new BlindFearProtectionStatus(insaneWiz));
+    
+    insaneWiz.CombatantChangeInterval = TimeSpan.FromSeconds(3.0);
+    insaneWiz.CombatantSearchStrategy = (attacker) => new RandomSearchStrategy();
+    
+    insaneWiz.Equip(new DragonScaleArmor());
+    
+    insaneWiz.Attacks = new CreatureAttackCollection()
+    {
+        new CreatureBasicAttack(20, 30, 69, new AttackDazeComponent (10), new AttackProneComponent(5))
+    };
+    
+    insaneWiz.Spells = new CreatureSpellCollection(25, 75)
```

Comment:
All of these will be instant spells without mana, if you are okay with it. Otherwise, need to pass in "instantCast: false" and use the mantra. 

Example: 

 { new CreatureSpell<MagicMissileSpell>(
            skillLevel: 11, cost: 5, 
            mantra: SpellHelper.GenerateMantra(), instantCast: false), 100, TimeSpan.Zero }